desktop/apps/sitemaps/index.coffee

```diff
@@ -6,17 +6,16 @@ app.set 'views', __dirname + '/templates'
 app.set 'view engine', 'jade'
 
 app.get '/robots.txt', routes.robots
-app.get '/news_sitemap.xml', routes.articles
-app.get '/video_sitemap.xml', routes.video
+app.get '/sitemap-news.xml', routes.news
+app.get '/sitemap-video.xml', routes.video
 app.get '/sitemap-misc.xml', routes.misc
 app.get '/sitemap-cities.xml', routes.cities
-app.get '/sitemap.xml', routes.index
```

Comment:
Yup. This is what is left in it:

```xml
<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
<sitemap>
<loc>https://www.artsy.net/sitemap-misc.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-cities.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-features-1.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-features-2.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-1.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-2.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-3.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-4.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-5.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-6.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-7.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-8.xml</loc>
</sitemap>
</sitemapindex>
```

I moved the ones still in use into robots.txt and fairs and features were rewritten as cinder jobs.